src/main/kotlin/uk/gov/gdx/datashare/config/MicrometerConfiguration.kt

```diff
@@ -24,17 +26,22 @@ class MicrometerConfiguration {
   fun getMeterRegistry(
     @Value("\${metrics.cloudwatch.namespace}") namespace: String,
     @Value("\${metrics.log-to-console:#{false}}") logToConsole: Boolean,
-  ): MeterRegistry? {
+  ): MeterRegistry =
     if (logToConsole) {
-      return LoggingMeterRegistry.builder(LoggingRegistryConfig.DEFAULT).loggingSink(log::debug).build()
+      LoggingMeterRegistry.builder(LoggingRegistryConfig.DEFAULT).loggingSink(log::debug).build()
+    } else {
+      CloudWatchMeterRegistry(
+        setupCloudWatchConfig(namespace),
+        Clock.SYSTEM,
+        cloudWatchAsyncClient(),
+      )
     }
-    val cloudWatchConfig: CloudWatchConfig = setupCloudWatchConfig(namespace)
-    return CloudWatchMeterRegistry(
-      cloudWatchConfig,
-      Clock.SYSTEM,
-      cloudWatchAsyncClient(),
-    )
-  }
+
+  @Bean
+  fun timedAspect(registry: MeterRegistry) = TimedAspect(registry)
```

Comment:
Why do we need this? Spring times controllers by default I thought: 
https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/aop/TimedAspect.java#L49
https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#actuator.metrics.supported.spring-mvc

Reply:
We need this for our percentiles. The other approach would be doing a meter filter. However that wouldn't then help with this case as micrometer times each and doesn't detect it is just an `/{id}`
![image](https://user-images.githubusercontent.com/47484139/219073053-79f62825-19b0-4cf7-9889-679ffddaec9a.png)

Reply:
For percentiles I have found a work around, but it doesn't help with the image above I think

Reply:
That's for our client calls to the LEV, which this doesn't affect. You can do `http.server.requests.count` with `{id}` just fine. If we want to collect aggregate stats on our client calls I'd rather tag the requests and use the default metrics I think

Reply:
Sure